ldes-fragmentisers/ldes-fragmentisers-common/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/fragmentation/repository/FragmentRepository.java

```diff
@@ -27,4 +27,6 @@ Optional<Fragment> retrieveMutableFragment(String viewName,
 	void removeLdesFragmentsOfView(String viewName);
 
 	void deleteTreeNodesByCollection(String collectionName);
+
+	List<Fragment> retrieveFragmentsByOutgoingRelation(LdesFragmentIdentifier fragmentId);
```

Comment:
What is "OutgoingRelation". I have to look at the implementation to get an idea. Maybe we can think of a more intuitive name? Or add javadoc to clarify this?

Reply:
Added javadoc